pkg/common/label.go

```diff
@@ -49,10 +49,13 @@ const (
 	// LabelNodePublishMothod is a pv label that indicates the method nodePuhlishVolume use
 	LabelNodePublishMothod = LabelAnnotationPrefix + "node-publish-method"
 
-	// AnnotationDataFlowAffinityInject is an annotation representing enabled the dataflow affinity injection
+	// AnnotationDataFlowAffinityInject is an annotation representing enabled the dataflow affinity injection, for internal use.
 	AnnotationDataFlowAffinityInject = LabelAnnotationPrefix + "dataflow-affinity.inject"
 	// LabelDataFlowAffinityPrefix is a prefix for customized dataflow affinity label name.
 	LabelDataFlowAffinityPrefix = "fluid.io."
```

Comment:
How about change the prefix to "dataflow-affinity.fluid.io."?

Reply:
ok, will fix.